templates/CRM/ManualDirectDebit/Form/InjectCustomGroup.tpl

```diff
@@ -19,59 +19,58 @@
   </div>
 </div>
 
-{literal}
 <script type="text/javascript">
-  CRM.$(function($) {
+  var directDebitID = '{$directDebitPaymentInstrumentId}';
+  var minimumDaysToFirstPayment = '{$minimumDaysToFirstPayment}';
+  {literal}
+
     moveDirectDebitMandateFormFields();
+    checkVisibilityOfMandateCustomGroup();
```

Comment:
The indentation here seems to be unconsistent. The rest of the code within the `{literal}{/literal}` tags is indented with two spaces less, so I guess we should remove them here (or add them there, whichever you prefer).